handbook/communication/company_meeting.md

```diff
@@ -1,6 +1,6 @@
 # Company meeting
 
-1. Schedule: every Monday 10:30-11:00am PST/PDT (link in the calendar event)
+1. Schedule: every Monday 8:00-8:30am PST/PDT (link in the calendar event)
```

Comment:
maybe we should use UTC to avoid shifts due to daylight savings?

We still have a lot of people in PST/PDT, is 8am maybe inconvenient due to its overlap with school start times? I guess its all a tradeoff, I miss plenty of company meetings currently due to overlap with trying to get my son to sleep.

Reply:
It's all a tradeoff. Based on people's feedback, I think this is the best for now. In general, we want it to be as early as possible for Californians (of which we have many), so that it's as non-late as possible for everyone else. I think 7am local time in California is too early, so pinning it to 8am California local time optimizes for early but not too early.